renderdoc/driver/vulkan/vk_msaa_array_conv.cpp

```diff
@@ -350,7 +349,7 @@ void VulkanDebugManager::CopyDepthTex2DMSToArray(VkImage destArray, VkImage srcM
     params.y = i % samples;    // currentSample;
     params.z = i / samples;    // currentSlice;
 
-    for(uint32_t s = 0; s < numStencil; s++)
+    for(uint32_t s = 0; s < 256; s++)
```

Comment:
This doesn't seem right to hardcode 256 in here for `numStencil`, since that doesn't make sense for depth images that have no stencil.